addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -13,6 +13,9 @@ const DynamicSnippet = publicWidget.Widget.extend({
     read_events: {
         'click [data-url]': '_onCallToAction',
     },
+    custom_events: {
+        's_dynamic_snippet_rerender': '_onRerender',
+    },
```

Comment:
It is indeed a indeed a fix. I targeted master with this one as I couldn't find a solution that could also work in stable. I also saw this fix as more of a "Nice to have" than a fix since it targets a debug-only snippet.

I can try again to find a way to make it work in stable if you want?

Reply:
Let's merge this here and we'll see in stable later. Can't the problematic button just be hidden in JS in the case it does not work maybe, something like that is always ok in stable (if fixed in master).